Packs/PagerDuty/.pack-ignore

```diff
@@ -3,7 +3,3 @@ ignore=IN126,IN145
 
 [file:README.md]
 ignore=RM106
-
-
-[file:PagerDuty_image.png]
-ignore=IM111
```

Comment:
did you remove that on purpose? build fails on that now.

Reply:
No I did not